src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalPrice = 40;
+  const discountForSevenDays = 50;
+  const discountForThreeDays = 20;
+  const rentalDays = days;
+  const totalCost = dailyRentalPrice * rentalDays;
+  let finalCost;
+
+  if (rentalDays < 3) {
+    finalCost = totalCost;
+  } else if (rentalDays >= 3 && rentalDays < 7) {
+    finalCost = totalCost - discountForThreeDays;
+  } else if (rentalDays >= 7) {
```

Comment:
According to checklist item 5, you should avoid using 'else' after 'if' with 'return'. However, since you are not using 'return' in the 'if' statements, consider restructuring the code to use 'if' with 'return' for each condition to simplify the logic.